### ISSUE TITLE
[SUGGESTION] Reverse Frame Stepping

### ISSUE THREAD

**Issue by Brittank88** [EclipseMenu/EclipseMenu#77]
I believe this mechanic was achieved by placing a 'checkpoint' at each frame step so that you can revert steps if necessary. A configurable 'length of history' would be good too (MH's reverse frame step lacked this and just cut off after a certain amount of backward steps).

**Comment by Vadym-beep** [comment]
I agree! There is already code for this in QOLMod, you can steal it from them
